### ISSUE TITLE
Game crashes after I win against five motionless Normal AIs

### ISSUE THREAD

**Issue by fusion809** [drogoganor/DarkReign#21]
Hi,

At the end of a game subject to #20 the game crashed on me. The logs are in (the latest of these exceptions is the exception log for this) [logs.zip](https://github.com/drogoganor/DarkReign/files/2479163/logs.zip). My OS is x86-64 Fedora 28, with the i3 window manager as my UI. `mono --version` returns:

```bash
Mono JIT compiler version 5.16.0.187 (tarball Mon Oct  8 08:52:21 UTC 2018)
Copyright (C) 2002-2014 Novell, Inc, Xamarin Inc and Contributors. www.mono-project.com
        TLS:           __thread
        SIGSEGV:       altstack
        Notifications: epoll
        Architecture:  amd64
        Disabled:      none
        Misc:          softdebug 
        Interpreter:   yes
        LLVM:          yes(3.6.0svn-mono-/)
        GC:            sgen (concurrent by default)
```

On this platform OpenRA Red Alert 20180923 runs fine (both the AppImage and it installed via my package manager), in case you're wondering whether my version of Mono or Lua is to blame.

Thanks for your time,
Brenton

**Comment by drogoganor** [comment]
Thanks for the detailed error report. I should have examined the logs after a full game because there's a lot of "Could not find collection" messages. As for the game crashing on victory, it seems a couple of wav files used for victory and defeat are not present in the demo content files. I've pushed a commit to solve both these issues.

**Comment by fusion809** [comment]
All is good, 'tis fixed now as of commit 30097d0990fa0f171f9c892ec9326e665e49e01c.
